server.py

```diff
@@ -21,8 +21,10 @@
 cors = CORS(app)
 
 
-API_KEY = os.environ.get("GPTNEO", None)
-headers = {"Authorization": API_KEY}
+BART = os.environ.get("BART", None)
+ROBERTA = os.environ.get("BART", None)
+headersBart = {"Authorization": BART}
+headersRoberta = {"Authorization": ROBERTA}
 MONGO_URI = os.environ.get("mongo_uri","mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/")

```

Comment:
yeah looking good. 
One doubt: 1) We aren't passing the headersRoberta anywhere right, do we need to declare one?

Reply:
It is for future purpose. We need that API token in future the way we needed it for bart. So just added as a provision.